src/main/resources/standardTestCases/dot_dir_stuck_running.test

```diff
@@ -0,0 +1,12 @@
+name: dot_dir_stuck_running
+testFormat: workflowsuccess
+
+files {
+  wdl: dot_dir_stuck_running/dot_dir_stuck_running.wdl
+}
+
+metadata {
+  workflowName: dot_dir_stuck_running
+  status: Succeeded
+  "outputs.dot_dir_stuck_running.out" : "hi"
```

Comment:
Good point, but I don't know how to verify file paths. Mainly listed the path as a visual debugging tool.